journal-writings/modernized/04-chapter-2.adoc

```diff
@@ -2,12 +2,12 @@
 
 [.chapter-synopsis]
 * His first journey, on a religious visit, into East Jersey, in company with Abraham Farrington
-* Thoughts on merchandizing, and learning a trade
+* Thoughts on merchandising, and learning a trade
```

Comment:
Because he's using this in an "archaic" way, I sort of think we should be more aggressive with this word and change it to `being a merchant`.

![2019-03-06_15-02-41](https://user-images.githubusercontent.com/7050938/53910104-0c8a4f00-4021-11e9-9b79-0490ad78c369.png)